packages/react/src/components/SkipLink/index.tsx

```diff
@@ -66,8 +67,11 @@ export default class SkipLink extends React.Component<
           onFocus={this.onFocus}
           onBlur={this.onBlur}
         >
-          <span className="SkipLink__item--first">{skipText}</span>&nbsp;
-          <span className="SkipLink__item--second">{targetText}</span>
+          <span className="SkipLink__item--first">{skipText}</span>
+          <span className="SkipLink__item--second">
+            <Icon type="triangle-right" aria-hidden="true" />
```

Comment:
```suggestion
            <Icon type="triangle-right" />
```

`aria-hidden` gets added automatically because `<Icon />` does not have a label: https://github.com/dequelabs/cauldron/blob/cea3659486f910cae66d2ab75e2e90616541b14d/packages/react/src/components/Icon/index.tsx#L47